f8a_worker/workers/sentiment_scorer.py

```diff
@@ -0,0 +1,324 @@
+"""
+Gathers component data from the graph database and aggregate the data to be presented
+by stack-analyses endpoint
+
+Output: TBD
+
+"""
+
+import os
+import json
+import requests
+
+from f8a_worker.base import BaseTask
+from f8a_worker.graphutils import GREMLIN_SERVER_URL_REST
+from f8a_worker.utils import get_session_retry
+
+import abc
+import os
+import datetime
+# import asyncio
+import time
+
+import requests
+import json
+from joblib import Parallel, delayed
+
+from google.cloud import language
+from google.cloud import bigquery
+
+
+class SentimentAnalyzer(object):
+    @abc.abstractmethod
+    def analyze_sentiment(self, text):
+        return
+
+
+class GoogleSentimentAnalyzer(SentimentAnalyzer):
+    def __init__(self, key_file_path):
+        os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = key_file_path
+        self.lang_client = language.Client()
+
+    def analyze_sentiment(self, text):
+        start_time = time.time()
+        print("Started analyzing sentiment")
```

Comment:
We could use proper logging here, but given time constrains, this will do just fine for now.